pkg/execution/planning.go

```diff
@@ -86,6 +87,9 @@ func (p *planningVisitor) EnterDocument(operation, definition *ast.Document) {
 	}
 
 	p.currentNode = p.currentNode[:0]
+	if len(p.planners) != 0 {
```

Comment:
Could u explain what we doing here?

Reply:
the `p.planners` slice gets modified during the visitor walking but the last planner is always the correct one - so we use the last one to plan the data source (if there exists one)